### DISCUSSION TITLE
Request: show dBV on Y axis

### DISCUSSION THREAD

**Discussion by mickbran** [mriscoc/Sparrow-Extended-UI-for-RIGOL-DHO800_DHO900#2] [Ideas]
Thanks for your application, it's very useful; however it would be even better if we couldsee the amplitude of the peaks, not just the frequency; so either add the Y axis to the chart or add the amplitude next to the frequency (buti t may get a bit cramped this way). 
Thank you.

**Comment by mriscoc** [comment]
I removed the Y axis labels to gain some screen space and because sometimes the values do not correspond to the acquired by the oscilloscope, but can be easily restored.

**Reply by mickbran** [reply]
That would be great, perhaps a button can be added to hide/unhide the Y axis.
Out of curiosity why would the values be different? What data is used to create the chart?
Many thanks.

**Reply by mriscoc** [reply]
What about this?
![image](https://github.com/mriscoc/RIGOL_DHO800_DHO900_GUI/assets/2745567/c5c65c8e-ee03-4759-a04d-62ff6456119d)

**Reply by mriscoc** [reply]
> Out of curiosity why would the values be different?

I checked that sometimes the Oscilloscope sent the data out of scale, but if the scale and offset of the FFT is set manually the correct scaled data is sent.

The data is retrieved using VISA commands: `:WAV:DATA?` in normal mode `:WAV:MODE NORMAL`

**Reply by mickbran** [reply]
it looks great, thank you.
